modules/angular-meteor-meteorCollection.js

```diff
@@ -28,15 +28,6 @@ angularMeteorCollections.factory('AngularMeteorCollection', ['$q', '$meteorSubsc
       function upsertObject(item, $q) {
         var deferred = $q.defer();
 
```

Comment:
Why did you remove it?  
Can you add it so that this will be in?
https://github.com/Urigo/angular-meteor/blob/master/modules/angular-meteor-meteorCollection.js#L31-L38